src/components/screens/ScoreSetEditor.vue

```diff
@@ -1282,22 +1285,21 @@ export default {
         this.secondaryPublicationIdentifiers = []
         this.publicationIdentifiers = []
         this.dataUsagePolicy = null
-        this.targetGene = null
-        this.targetGenes = []
-        this.referenceGenome = null
-        this.assembly = null
-        this.assemblySuggestions = []
-        this.assemblyDropdownValue = null
-        this.geneName = null
-        this.geneNameAccessionSuggestions = []
-        this.geneNameDropdownValue =  null
-        this.targetGenes = []
         this.extraMetadata = {}
+        this.resetTarget()
       }
     },
 
     resetTarget: function () {
+      this.assembly = null
+      this.assemblySuggestions = []
+      this.assemblyDropdownValue = null
+      this.existingTargetGene = null
+      this.geneName = null
+      this.geneNameAccessionSuggestions = []
+      this.geneNameDropdownValue =  null
       this.fileCleared('targetGeneTargetSequenceSequenceFile')
+      this.referenceGenome = null
       this.targetGene = {
```

Comment:
please also fix redundancy with l686 by adding helper function returning this target gene

Reply:
it looks like you're not yet using the helper function for the initialization at l686.

Reply:
Thanks Ashley! Now I know what is the helper function.